client-spark/spark3/src/main/java/org/apache/spark/shuffle/RssShuffleManager.java

```diff
@@ -1187,6 +1148,94 @@ public synchronized boolean reassignShuffleServers(
     }
   }
 
+  @Override
+  public ShuffleServerInfo reassignFaultyShuffleServer(
```

Comment:
How about naming `reassignFaultyShuffleServerForTasks`

Reply:
`reassignFaultyShuffleServerForTasks` is better, I'll rename it.